packages/next-on-pages/templates/cache/adaptor.ts

```diff
@@ -203,7 +203,13 @@ export class CacheAdaptor {
 	 * @returns The fully-formed cache key for the suspense cache.
 	 */
 	public buildCacheKey(key: string) {
-		return `https://${SUSPENSE_CACHE_URL}/entry/${key}`;
+		// Note: we use the pages commit sha if present so that caches from
+		//       a previous deployment are not reused for new ones
```

Comment:
Next.js docs state that the cache should be shared across deployments.
 
<img width="672" alt="image" src="https://github.com/cloudflare/next-on-pages/assets/10815538/4e3a0c39-8d8c-4809-a05c-6e9f5f96a550">

Reply:
oh.... thanks I totally missed that 😓